include/geode/model/representation/builder/detail/copy.h

```diff
@@ -42,80 +45,80 @@ namespace geode
 {
     namespace detail
     {
-        using Mapping = absl::flat_hash_map< uuid, uuid >;
+        using UUIDMapping = BijectiveMapping< uuid >;
```

Comment:
Why UUID and not uuid?

Reply:
I do not know. uuidMaping? UuidMapping?

Reply:
uuidMapping or something else like ComponentMapping